pkg/controller/clone-controller.go

```diff
@@ -360,6 +373,20 @@ func (c *CloneController) processPvcItem(pvc *v1.PersistentVolumeClaim) error {
 	return nil
 }
 
+func (c *CloneController) setOrIncrementExpectation(pvcKey string) error {
```

Comment:
Might be interesting to push this into RaiseExpectation, but that opens up a problem if someone raises the wrong key, and it will simply create the expectation instead of failing. But something to think about none the less.